editor/src/clj/editor/atlas.clj

```diff
@@ -633,6 +639,8 @@
 
   (output texture-image    g/Any               (g/fnk [packed-image texture-profile]
                                                  (tex-gen/make-preview-texture-image packed-image texture-profile)))
+  
+  (output texture-set-pb   g/Any               produce-atlas-texture-set-pb)
```

Comment:
This is required in order to read the final texture set output.
(I.e. we read it with dmDDF::LoadMessage() in the spine extension)